SonarQube.MSBuild.Tasks/Targets/SonarQube.Integration.targets

```diff
@@ -351,4 +351,36 @@
   <!-- End of FxCop section-->
   <!-- **************************************************************************** -->
 
+
+  <!-- **************************************************************************** -->
+  <!-- StyleCop -->
+  <!-- **************************************************************************** -->
+  <!-- Set the analysis properties required by the StyleCop plugin. The actual execution of 
+       StyleCop is handled by the plugin. -->
+  <Target Name="SetStyleCopAnalysisSettings"
+        Condition=" $(RunSonarQubeAnalysis) == 'true' "
```

Comment:
Done. I've updated the targets file and removed all references in the tests to $(RunSonarQubeAnalysis).